openvdb_points/tools/PointConversion.h

```diff
@@ -728,6 +772,57 @@ struct ConvertPointDataGridGroupOp {
     const bool                              mInCoreOnly;
 }; // ConvertPointDataGridGroupOp
 
+inline bool voxelSizeFromVolume(const double volume, const size_t estimatedVoxelCount, float& voxelSize)
```

Comment:
Just so this little function isn't isolated in a completely different place in the code, might be good to include it as a static member of a Local struct nested in the autoVoxelSize function? Lots of example of this pattern in VDB, but here's one - Transform::print(...)